lib/executor/coredump.cpp

```diff
@@ -0,0 +1,219 @@
+#include "executor/coredump.h"
+#include "ast/section.h"
+#include "common/errcode.h"
+#include "common/types.h"
+#include "loader/serialize.h"
+#include "runtime/stackmgr.h"
+#include "spdlog/spdlog.h"
+#include <cstdint>
+#include <cstring>
+#include <vector>
+
+#define ENCODE_LEB128(Content, Value)                                          \
```

Comment:
You included the `serialize.h`.
Why not just use the `serializeUN()` or `serializeU32()`?

Reply:
Yes, you can export them from the serializer.